### PR TITLE
Fix master and fix integration tests lifecycles

### PR DESCRIPTION
**Pull request by michel-tricot**
## What
Master is broken because we are not compiling integration tests when we `build`. This PR fixes the broken build and ensure we have the correct lifecycle.
Also fixes an invalid image name in postgres source

### CODE REVIEW

File: dataline-integrations/singer/bigquery/destination/build.gradle [hunk 1]
```diff
@@ -1,34 +1,51 @@
+import com.bmuschko.gradle.docker.tasks.image.DockerBuildImage
```
Comment (michel-tricot) [inline dataline-integrations/singer/bigquery/destination/build.gradle:1]: Going back on a past comment I did, it is actually really anoying to have imports sprinkled in the code.

File: dataline-integrations/singer/bigquery/destination/build.gradle [hunk 2]
```diff
@@ -1,34 +1,51 @@
...
+    integrationTest {
+        java {
+            srcDir 'src/test-integration/java'
+        }
+        resources {
+            srcDir 'src/test-integration/resources'
+        }
+    }
+}
+test.dependsOn('compileIntegrationTestJava')
```
Comment (michel-tricot) [inline dataline-integrations/singer/bigquery/destination/build.gradle:20]: That is the fix to ensure we build the integration tests. @jrhizor if you think there is a better way, let me know.

**Review by michel-tricot — COMMENTED** [review]
I wonder if we can use some kind of a plugin for these configs. it's really annoying to copy an paste

**Review by sherifnada — APPROVED** [review]

File: dataline-integrations/singer/bigquery/destination/build.gradle (hunk 2 above)
Comment (jrhizor) [reply dataline-integrations/singer/bigquery/destination/build.gradle:20]: Looks good

**Pull request merged**